clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -70,6 +70,46 @@
         }                                      \
     } while (0)
 
+int
+instruction_converter_t::write_thread_exit(byte *buffer, thread_id_t tid)
+{
+    online_instru_t instru(NULL, false, NULL);
+    return instru.append_thread_exit(buffer, tid);
+}
+int
+instruction_converter_t::write_marker(byte *buffer, trace_marker_type_t type,
+                                      uintptr_t val)
+{
+    online_instru_t instru(NULL, false, NULL);
```

Comment:
It seems wasteful to construct one of these every single time: is there a way to have a static copy or sthg.

Reply:
True, I can have a global static in the .cpp file.